libs/wingsdk/src/fs/fs.ts

```diff
@@ -17,7 +17,23 @@ export interface ReadFileOptions {
   readonly encoding?: BufferEncoding;
   /**
    * The `flag` can be set to specify the attributes.
-   * @default 'r'.
+   * @default `r``.
```

Comment:
Let's keep consistency where we can - seems like we usually use double quotes for defining `@default`:
```suggestion
   * @default "r".
```

Reply:
Thanks for reporting this, it was actually a typo when I copy-pasted the existing code.